src/content/developers/tutorials/hello-world-smart-contract.index.md

```diff
@@ -0,0 +1,362 @@
+
+```
+npx hardhat run scripts/deploy.js --network ropsten
+```
+
+You should then see something like:
+
+```
+Contract deployed to address: 0x6cd7d44516a20882cEa2DE9f205bF401c0d23570
+```
+
+If we go to the [Ropsten etherscan](https://ropsten.etherscan.io/) and search for our contract address we should able to see that it has been deployed successfully. The transaction will look something like this:
+
+![etherscan contract](https://user-images.githubusercontent.com/23513719/111497395-12869080-870f-11eb-8fdb-9bedb403bb01.png)
+
+The `From` address should match your Metamask account address and the To address will say “Contract Creation” but if we click into the transaction we’ll see our contract address in the `To` field:
+
+![etherscan transaction](https://user-images.githubusercontent.com/23513719/111497426-187c7180-870f-11eb-87cc-fa1000afd90c.png)
+
+Congrats! You just deployed a smart contract to the Ethereum chain 🎉
+
+To understand what’s going on under the hood, let’s navigate to the Explorer tab in our [Alchemy dashboard](https://dashboard.alchemyapi.io/explorer). If you have multiple Alchemy apps make sure to filter by app and select “Hello World”.
+![hello world explorer](https://user-images.githubusercontent.com/23513719/111497637-42359880-870f-11eb-9ce3-b8645b8e4766.png)
+
+Here you’ll see a handful of JSON-RPC calls that Hardhat/Ethers made under the hood for us when we called the `.deploy()` function. Two important ones to call out here are [`eth_sendRawTransaction`](https://docs.alchemyapi.io/alchemy/documentation/alchemy-api-reference/json-rpc#eth_sendrawtransaction), which is the request to actually write our contract onto the Ropsten chain, and [`eth_getTransactionByHash`](https://docs.alchemyapi.io/alchemy/documentation/alchemy-api-reference/json-rpc#eth_gettransactionbyhash) which is a request to read information about our transaction given the hash (a typical pattern when 
+transactions). To learn more about sending transactions, check out this tutorial on [sending transactions using Web3](https://docs.alchemyapi.io/alchemy/tutorials/sending-transactions-using-web3-and-alchemy)
+
+That’s all for part 1 of this tutorial, in part 2 we’ll actually [interact with our smart contract](https://docs.alchemyapi.io/alchemy/tutorials/hello-world-smart-contract#part-2-interact-with-your-smart-contract) by updated our initial message, and in part 3 we’ll [publish our smart contract to Etherscan](https://docs.alchemyapi.io/alchemy/tutorials/hello-world-smart-contract#optional-part-3-publish-your-smart-contract-to-etherscan) so everyone will know how to interact with it.
+
+**Want to learn more about Alchemy? Check out our [website](https://alchemyapi.io/eth). Never want to miss an update? Subscribe to our newsletter[here](https://www.alchemyapi.io/newsletter)! Be sure to also follow our [Twitter](https://twitter.com/alchemyplatform?lang=en) and join our [Discord](https://discord.com/invite/u72VCg3)**.
```

Comment:
Little confused at the end here... Are there additional tutorials coming? @ryancreatescopy @samajammin thoughts on teeing up one tutorial locally to feed into all Alchemy links?